<ui version="4.0" >
 <class>CustomAlphaBlend</class>
 <widget class="QDialog" name="CustomAlphaBlend" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>94</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>94</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Custom Alpha Blend</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lblTitle" >
     <property name="text" >
      <string>Select the alpha blend value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lbl01percent" >
       <property name="text" >
        <string>1%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hsAlphaBlend" >
       <property name="whatsThis" >
        <string>Alpha blend value (%)</string>
       </property>
       <property name="accessibleName" >
        <string>Alpha blend value slider</string>
       </property>
       <property name="accessibleDescription" >
        <string>Alpha blend value (%)</string>
       </property>
       <property name="minimum" >
        <number>0</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="value" >
        <number>50</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl100percent" >
       <property name="text" >
        <string>100%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblCurrentValue" >
       <property name="text" >
        <string>Current: &lt;b>0.1%&lt;/b></string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>101</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="whatsThis" >
        <string>Set the selected alpha blend value and close this dialog</string>
       </property>
       <property name="accessibleName" >
        <string>Ok button</string>
       </property>
       <property name="accessibleDescription" >
        <string>Set the selected alpha blend value and close this dialog</string>
       </property>
       <property name="text" >
        <string>Ok</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="whatsThis" >
        <string>Close this dialog without changes</string>
       </property>
       <property name="accessibleName" >
        <string>Cancel button</string>
       </property>
       <property name="accessibleDescription" >
        <string>Close this dialog without changes</string>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
